docs/components/defining-bundles.md

```diff
@@ -0,0 +1,87 @@
+# Defining Bundles
+
+A bundle is defined in a dedicated package named `comp/<bundleName>`.
+The package must have the following defined in `bundle.go`:
+
+ * Extensive package-level documentation.
+   This should define:
+
+     * The purpose of the bundle
+     * What components are and are not included in the bundle.
+       Components might be omitted in the interest of binary size, as discussed in the [overview](./components.md).
+     * Which components are automatically instantiated.
+     * Which other _bundles_ this bundle depends on.
+       Bundle dependencies are always expressed at a bundle level.
+
+ * A team-name comment of the form `// team: <teamname>`.
+   This is used to generate CODEOWNERS information.
+
+ * `componentName` -- the Go path of the component, relative to the repository root, e.g., `comp/core/health`.
```

Comment:
Should we get this automatically through [`runtime.Caller`](https://pkg.go.dev/runtime@go1.18.6#Caller)?

Reply:
Good idea -- fixed in de7a845d7cccbb0d92d09403def7791c756420c9